src/Nuvem.PharmacyManagementSystem.Pharmacy.Api/Controllers/PharmacyController.cs

```diff
@@ -59,9 +45,9 @@ public async Task<ActionResult<PharmacyModel>> GetPharmacyById(int id)
         [SwaggerResponse((int)HttpStatusCode.OK)]
         [SwaggerResponse((int)HttpStatusCode.BadRequest)]
         [SwaggerResponse((int)HttpStatusCode.NotFound)]
-        public async Task<ActionResult<PharmacyModel>> UpdatePharmacyById(int id, [FromBody]PharmacyModel pharmacyModel)
+        public async Task<ActionResult> UpdatePharmacyById(int id, [FromBody]PharmacyModel pharmacyModel)
```

Comment:
Earlier this week when you had a doctor's appt, James suggested just relying on the id in the pharmacyModel here instead of having a separate parameter for id

Reply:
I will do the change. Frankly speaking, I was wondering too if there's a need for separate id parameter. Thanks for pointing it out.